### ISSUE TITLE
Deprecation Warnings from @material

### ISSUE THREAD

**Issue by tuckergordon** [hperrin/svelte-material-ui#281]
**Describe the bug**
Using themed components results in deprecation warnings from @material (e.g. `Using / for division is deprecated`). This can be fixed by upgrading to the latest version of @material (as of writing, that means upgrading from 11 to 12). 

**To Reproduce**
1. Create a new Svelte project 
```
npx degit sveltejs/template my-svelte-project`)
```
2. Install smui/button
```
npm install --save @smui/button
```
3. Go through the steps for the [Advanced Styling Method](https://sveltematerialui.com/SASS.md#advanced-styling-method)
  a. `npm install --save-dev sass rollup-plugin-postcss`
  b. Add sass to your rollup config
  c. Add a blank `_smui-theme.scss` file (this prevents a compilation error, but it doesn't need any code in it)
4. Add a styled button to your App component:
```html
<script>
  import Button from '@smui/button/styled';
</script>

<Button>Hello</Button>
```
5. `npm run dev` will result in deprecation warnings

**Screenshots**
![image](https://user-images.githubusercontent.com/7843139/132344460-04c1d563-2bd1-4e87-89f5-cb68c2f9fdbb.png)

**Additional context**

I've only tested this with `Button`, but manually installing the latest version of the material dependencies fixed the issue. E.g. installing
```
    "@material/button": "^12.0.0",
    "@material/elevation": "^12.0.0",
    "@material/feature-targeting": "^12.0.0",
    "@material/ripple": "^12.0.0",
    "@material/theme": "^12.0.0",
```
But given how many components there are, I wouldn't be surprised if others required more adjustments to work with the newer version of material.

**Comment by tuckergordon** [comment]
Also this is probably a separate issue, but using theming and themed components results in much slower build times. My guess is that it's related to the @material theming. Angular ran into this issue - [here](https://github.com/angular/components/issues/22676) and [here](https://github.com/angular/angular-cli/issues/20713)

**Comment by tuckergordon** [comment]
Also this is probably a separate issue, but I'm noticing slow build times with smui components. My guess is that it's related to the issues that Angular material also ran into - [here](https://github.com/angular/components/issues/22676) and [here](https://github.com/angular/angular-cli/issues/20713)

**Comment by tuckergordon** [comment]
Whoops sorry, forgot to follow up. Yes, this was closed by #350
